src/assets/javascripts/templates/search/index.tsx

```diff
@@ -93,9 +94,21 @@ function renderSearchDocument(
             {truncate(document.text, 320)}
           </p>
         }
-        {document.tags && document.tags.map(tag => (
-          <span class="md-tag">{tag}</span>
-        ))}
+        {document.tags && (
```

Comment:
I guess we could see about supporting tags?

Reply:
I'm leaving this as is for now. If someone asks, then we can discuss it again (or just create a new issue as a reminder).